packages/meditrak-app/README.md

```diff
@@ -1,176 +1,88 @@
-# @tupaia/meditrak
+# Prerequisites
 
-Android and iOS app for surveying medical facilities
+- nvm use
```

Comment:
I had to add an `.nvmrc` file with `16.20.2` in it to the root of the `meditrak-app` package